src/components/Avatar/index.test.tsx

```diff
@@ -37,38 +37,37 @@ describe('grabInitials', () => {
 describe('Avatar component', () => {
   it('renders name', () => {
     render(<Avatar name="Joey Tribbiani" />);
-    expect(screen.getByTitle('Joey Tribbiani', { name: 'JT' })).toBeTruthy();
+    expect(screen.getByTitle('Joey Tribbiani')).toHaveTextContent('JT');
```

Comment:
I'm curious what you found out here that required you to migrate to this approach, as we're using things similar to what was used before in `src/components/Button/index.test.tsx`

Reply:
Ah! The power of TypeScript, those errors were in fact legit.